addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -352,6 +352,7 @@ body.editor_enable.editor_has_snippets {
     > #o_scroll {
         background-color: $o-we-sidebar-blocks-content-bg;
         padding: 0 $o-we-sidebar-blocks-content-spacing;
+        height: 100%;
```

Comment:
Maybe add a comment to explain why this is needed, something like "give enough space for tips pointing at snippets after a snippet search"